### PR TITLE
 feat: support Chrome 63

### PR DESCRIPTION
**Pull request by aslushnikov**
Puppeteer needs to be mostly functional with stable chrome. The first
stable chrome that will be supported will be Chrome 63.

This patch:
- adds backwards compatibility with Chrome 63 (currently, it's only the
  renamed request interception protocol call)
- whitelists a few tests to be ran only on Chrome 64+
- adds a new `unit-chrome-63` command that runs unit tests against
  Chrome M63
- adds a travis job to run tests against Chrome 63.

Over time, when Chrome 63 will be promoted to beta, we'll switch to
support Chrome Beta, and eventually Chrome Stable.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "unit": "jasmine test/test.js",
+    "unit-chrome-63": "node ./utils/downloader.js --download 508578 && CHROME=$(node ./utils/downloader.js --path 508578) npm run unit",
```
Comment (JoelEinbinder) [inline package.json:12]: Use cross env for windows support

**Review by JoelEinbinder — APPROVED** [review]

**Comment by aslushnikov** [comment]
Closing this since we missed the chrome 63 target for now.

**Comment by Bnaya** [comment]
Any idea when its going to be back?

**Comment by aslushnikov** [comment]
@Bnaya we're aiming for M64 now.

**Pull request closed without merging**